style.css

```diff
@@ -530,7 +533,232 @@ ul {
+  padding: 15px 30px 40px 30px;
+  z-index: 100;
+  backdrop-filter: blur(0.31rem);
+  margin: 30px;
+  overflow-y: scroll;
+  box-shadow: 10px 10px 29px 0 rgba(0, 0, 0, 0.75);
+}
+
+body.active {
+  overflow: hidden;
+}
+
+.desktop-version.active {
+  filter: blur(0.3rem);
+  transition: 0.3s;
+}
+
+.image-container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 75vh;
+  border-radius: 8px;
+  background-color: #e4e7e5;
+}
+
+.modal-image {
+  width: 85%;
+  height: 75vh;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-header {
+  display: flex;
+  padding: 10px 10px;
+  justify-content: space-between;
+}
+
+.modal-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.modal-list {
+  list-style: disc;
+  display: flex;
+  margin: 5px 0 20px 5px;
+}
+
+.close-modal-button {
+  background: transparent;
+  border: none;
+  color: #67798e;
+  font-size: 28px;
+}
+
+.modal-list li {
+  margin-right: 25px;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.modal-list li:nth-child(1) {
+  list-style-type: none;
+  color: #344563;
+}
+
+.modal-content-detail {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  column-gap: 25px;
+}
+
+.modal-detial {
+  margin-top: 30px;
+  padding: 5px;
+}
+
+.modal-content-detail .modal-content-list .modal-language {
+  margin: 15px;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.modal-content-detail .modal-content-list .modal-language li {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 10px;
+  width: fit-content;
+  padding: 1px 15px;
+  margin-right: 15px;
+  margin-top: 15px;
+}
+
+hr {
+  margin: 25px 20px 35px 20px;
+  border: 1px solid #ebecf0;
+  font-size: 3px;
+  width: 70%;
+}
+
+.model-links {
+  display: flex;
+  width: 20rem;
+  margin: 20px;
+  justify-content: space-evenly;
+}
+
+.modal-link-btns {
+  display: flex;
+  flex-wrap: wrap;
+  width: fit-content;
+  justify-content: center;
+  align-content: center;
+  align-items: end;
+  text-align: center;
+  background: #fff;
+  padding: 9px 20px;
+  border: 1px solid #6070ff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 12px;
+  color: #6070ff;
+}
+
 @media (max-width: 768px) {
+  /* model-start */
+
+  .modal {
+    overflow-x: hidden;
+    width: 100%;
+    height: 100vh;
+    max-width: 80%;
+    padding: 15px;
+  }
+
+  .modal-content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .image-container {
+    height: 40vh;
+  }
+
+  .modal-image {
+    width: 100%;
+    height: 40vh;
+    align-self: center;
+  }
+
+  .modal-title {
+    font-size: 30px;
+    line-height: 32px;
+  }
+
+  .modal-list {
+    list-style: disc;
+    display: flex;
+    margin: 5px 0 10px 10px;
+  }
```

Comment:
- [ ] Kindly fix the alignement for the tchnologies since it is aligned beside each other in the figma but it is different in you design 💪🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/180224391-dcdc58e0-3bee-48e4-b7f2-d6c0da22627b.png)


- Figma design
![image](https://user-images.githubusercontent.com/58553711/180224497-b91c9c37-992a-47e9-9fce-b264ca130612.png)